tutorials/abap-extensibility-cbo-execute-outbound-service/abap-extensibility-cbo-execute-outbound-service.md

```diff
@@ -130,286 +128,267 @@ In order to allow communication with the SAP API Business Hub Sandbox you have t
+6. **Save** the Arrangement.
 
 
-### Extend custom business object with feedback fields
+### Extend custom business object data structure
 
 Add fields to persists feedback at the custom business object `Bonus Entitlement`.
 
-1. Go to **Home** of your SAP S/4HANA system's Fiori Launchpad.
-
-    ![Fiori Home Button](FioriHomeButton.png)
+1. Start typing **Custom Business Objects** in the Launchpad search and open the App from the results.
 
-2. Open **Custom Business Objects** application in **Extensibility** launchpad group.
+    ![Custom Business Objects application from search results](FLP_search_resultCBO.png)
 
-    ![Custom Business Objects application tile](tile_CBO.png)
+2. Open the business object `Bonus Entitlement`.
+   
+3. Start Edit Mode by executing the **Edit Draft** action.
 
-3. Open the business object `Bonus Entitlement`.
+4. Switch to **Fields** section.
 
-4. Start Edit Mode by executing the **Edit Draft** action.
-
-5. Switch to **Fields** section.
-
-6. Add following **New** fields
+5. Add following **New** fields
 
     | Field Label | Field Identifier | Field Type | Field Properties |
     | :---------- | :--------------- | :----------| :----------------|
     | **`Feedback`**| **`Feedback`** | **`Text`** | Length: **`255`** |
     | **`Feedback's language`** | **`FeedbacksLanguage`** | **`Text`** | Length: **`2`** |
     | **`Feedback in english`** | **`FeedbackInEnglish`** | **`Text`** | Length: **`255`** |
 
-7. **Publish** the business object.
+6. **Publish** the business object.
 
 
-### Open custom business object logic
+### Enhance custom business object logic
 
 Now as the business object has just been published, the logic can be enhanced by the translation functionality. ABAP for key users was enhanced by the classes `CL_BLE_HTTP_CLIENT`, `CL_BLE_HTTP_REQUEST` and `CX_BLE_HTTP_EXCEPTION` to enable you to work with HTTP requests.
 
 1. Switch to **Logic** section.
 
-    ![Switch to Logic section](CBO_LogicSection.png)
-
 2. Enter the **After Modification** Event Logic.
 
     ![Enter After Modification logic](CBO_go2AfterModify.png)
 
+3. Code HTTP client creation
+   In order to call an external service from there you need to create an HTTP client in your custom business object logic.
 
-### Code HTTP client creation
-
-
-1. In the already existing coding go to the end but before this snippet
-
-    ```abap
-        ELSE.
-            RETURN.
-        ENDIF.
-    ENDIF.
-    ```
-2. Implement a check if the outbound service is available
-
-    ```abap
-    * Check if the outbound service is available
-    CHECK cl_ble_http_client=>is_service_available(
-        communication_scenario = 'YY1_MACHINE_TRANSLATION_API'
-        outbound_service       = 'YY1_OS_MACHINE_TRANSLATION_API_REST'
-    ) = abap_true.
-    ```
-3. Implement creation of HTTP client
+    - In the already existing coding go to the end but stay in front of the last ENDIF
+  
+    - Implement a check if the outbound service is available
```

Comment:
re-phrasing suggestion: Implement the following code to check whether the outbound service is available

Reply:
well, it shall somehow still be a task to try doing yourself not just copy and paste

Reply:
Good point, then maybe you could re-phrase as: Implement the code to check whether the outbound service is available.
And then you could put the code in a hint box. Or write something like 'It should look something like this: CODE BLOCK HERE'

Reply:
but its like that at many more places. I will simply leave it like that.